pylint_odoo/checkers/no_modules.py

```diff
@@ -226,6 +226,12 @@
         'print-used',
         settings.DESC_DFLT
     ),
+    'W%d20' % settings.BASE_NOMODULE_ID: (
+        'Translation method _(%s) is using positional string printf formatting. '
+        'Use named placeholder "_(%%(placeholder)s)" instead.',
```

Comment:
```suggestion
        'Use named placeholder `_("%%(placeholder)s")` instead.',
```

I guess the double-percent symbol is needed, right?

Reply:
Done